app/api/orders.py

```diff
@@ -208,9 +208,11 @@ def after_create_object(self, order, data, view_kwargs):
                 order.completed_at = datetime.utcnow()
 
             order_url = make_frontend_url(path='/orders/{identifier}'.format(identifier=order.identifier))
-            for organizer in order.event.organizers:
+            for organizer, coorganizer in zip(order.event.organizers, order.event.coorganizers):
```

Comment:
@mrsaicharan1 Consider the case:
```
organiser = [user1]
co-organiser = []
```
in this case, neither organiser not co-organiser will receive the notifications. Maybe employ this in separate loops?

Reply:
@uds5501 Got it. Changing it rn.

Reply:
@uds5501 Got it.Changed it accordingly